src/meta/src/manager/streaming_job.rs

```diff
@@ -73,120 +97,132 @@ impl StreamingJob {
         match self {
             Self::MaterializedView(table) => table.id = id,
             Self::Sink(sink) => sink.id = id,
-            Self::Table(_, table) => table.id = id,
+            Self::Table(_, table, ..) => table.id = id,
             Self::Index(index, index_table) => {
                 index.id = id;
                 index.index_table_id = id;
                 index_table.id = id;
             }
+            StreamingJob::Source(_) => {
+                // do nothing
```

Comment:
Why do nothing as the behavior of `id` seems to be normal? Please elaborate more or mark it as `unreachable`.